apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-launch.tsx

```diff
@@ -88,13 +111,22 @@ function WelcomeTour( { siteIntent }: { siteIntent?: string } ) {
 		const paymentBlockIndex = tourSteps.findIndex( ( step ) => step.slug === 'payment-block' );
 		tourSteps.splice( paymentBlockIndex, 1 );
 	}
-	const { isInserterOpened, isSidebarOpened, isSettingsOpened } = useSelect( ( select ) => ( {
-		isInserterOpened: select( 'core/edit-post' ).isInserterOpened(),
-		isSidebarOpened: select( 'automattic/block-editor-nav-sidebar' )?.isSidebarOpened() ?? false, // The sidebar store may not always be loaded.
-		isSettingsOpened:
-			select( 'core/interface' ).getActiveComplementaryArea( 'core/edit-post' ) ===
-			'edit-post/document',
-	} ) );
+	const { isInserterOpened, isSidebarOpened, isSettingsOpened } = useSelect(
+		( select ) => ( {
+			isInserterOpened: (
+				select( 'core/edit-post' ) as CoreEditPostPlaceholder
+			 ).isInserterOpened(),
+			isSidebarOpened:
+				(
+					select( 'automattic/block-editor-nav-sidebar' ) as WpcomBlockEditorNavSidebarSelectors
```

Comment:
`as WpcomBlockEditorNavSidebarSelectors` doesn't account for the store being missing